scripts/updateDeployments.ts

```diff
@@ -65,79 +66,92 @@ export async function updateBalancerDeployments() {
     const data: Record<string, SupportedNetworkResponse> = await res.json();
 
     const supportedNetworks = Object.entries(data).map(
-        ([name, { chainId }]) => ({ name, chainId }),
+        ([name, { chainId }]) => ({
+            name,
+            chainId,
+            deploymentsUrl: `https://raw.githubusercontent.com/balancer/balancer-deployments/refs/heads/master/addresses/${name}.json`,
+        }),
     );
 
+    // Manually add sonic since it live in seperate beets org repo
+    supportedNetworks.push({
+        name: 'sonic',
+        chainId: sonic.id,
+        deploymentsUrl:
+            'https://raw.githubusercontent.com/beethovenxfi/balancer-deployments/refs/heads/master/addresses/sonic.json',
+    });
```

Comment:
This is much better than the foolish way I was duplicating code in the base PR :sunglasses: